packages/components/src/internal/components/editable/models.ts

```diff
@@ -690,12 +688,12 @@ export class EditorModel
     }
 
     static convertQueryDataToEditorData(
-        data: Map<string, any>,
-        updates?: Map<any, any>,
+        data: Map<string, any>, // this map is keyed by column name
```

Comment:
nit: These types of comments are useful but as discussed an actual type would go a lot further in enforcing these things. I think we should look to add a type for field keys in the near future.

Reply:
yes, I agree. I have it on my TODO list. I hope to get to it soon (but as a separate PR)